phpseclib/Crypt/DES.php

```diff
@@ -140,24 +140,13 @@ class Crypt_DES extends Crypt_Base
     var $block_size = 8;
 
     /**
-     * The Key
+     * Key Length
```

Comment:
key length vs key size - might be useful to be consistent as to not confuse ourselves.